content/doc/book/managing/spawning-processes.adoc

```diff
@@ -141,9 +142,10 @@ if (outputFile != "") {
 // WScript.Echo(exeStr);
 // WshShell.Run(exeStr);
 WshShell.Run(exeStr, windowStyle, false);
-....
+----
 
-....
+[source]
+----
```

Comment:
```suggestion
[source]
----
```
So this is contradicting the first example given, where the first method includes:
`<exec executable="at">
  <arg value="${next-minute}" />
  <arg value="/interactive" />
  <arg value="${jboss.home}\bin\run.bat" />
</exec>`

This separates the execution call from the method being provided, and since there is no explanation as to what that is (in this change) or why it's presented in its own code block, it should not be separated out like that.

Here, it is included:
<img width="1017" alt="included execution call" src="https://github.com/jenkins-infra/jenkins.io/assets/99040580/51d7b357-8db3-40c3-983a-d0a0c986ccd3">

Here it is not:
<img width="988" alt="separated execution call" src="https://github.com/jenkins-infra/jenkins.io/assets/99040580/2757d5aa-0dd3-4a49-996e-85d235fb2fd5">

This is how the page is displayed currently:
<img width="785" alt="current view" src="https://github.com/jenkins-infra/jenkins.io/assets/99040580/493e0a6c-ff62-4fb5-9cf9-e8a21381ab3b">

Also, you can always use [the deploy site](https://deploy-preview-6794--jenkins-io-site-pr.netlify.app/doc/book/managing/spawning-processes/) to preview how the changes look, if the local clone is not necessarily rendering properly.

Reply:
For context:
I am just fixing up a commit, strongly assuming its changes are (unintentionally) incomplete.
In the commit that I am fixing up, it doesn't seem that the code blocks were _intentionally_ concatenated, hence calling this a _fixup_ to split them again. It seems that the author thought it was a single big code block, overlooking that there are multiple ones.
But also independent of that, I can explain why I think the separation of code blocks, which I do in this PR, makes more sense.

---

> This separates the execution call from the [wrapper] method being provided

Correct, the reader must store them in different files anyway, due to being different languages (one JS, the other Ant XML).

> So this is contradicting the first example given, where the first method includes [the call]

The option with the 'at' command only consists of a single call, right, no further self-provided wrapper script, hence no code blocks split there.

The option "antRunAsync.js" involves a self-provided wrapper, which is called by Ant. Therefore, it makes sense to split the code block of the wrapper from the call to it. This is not a _contradiction_ between the 2 options, just different approaches.

Call chain comparison (for understanding):
option "'at' command": Ant -> **at.exe** -> daemon
option "antRunAsync.js": Ant -> **cscript.exe -> antRunAsync.js** -> daemon

---

> there is no explanation as to what that is (in this change) or why it's presented in its own code block

True, although it should now (from the above reasons) be apparent that not separating the code blocks at all is still worse than at least separating them, _even if_ without explanation for now. I added an explanation.